vault/dendron.topic.schema.md

```diff
@@ -266,5 +266,5 @@ Common errors:
 
 ### Testing schemas
 
-If you want to test what your schemas will match, you can use a glob tester like [Glob teste ](https://globster.xyz/) to do so
```

Comment:
I _think_ this was missing the 'r'...  If this is intended, lmk and I can revert this one.